### PR TITLE
www/nginx: add authentication via TLS/Basic

### PR DESCRIPTION
**Pull request by fabianfrz**

### CODE REVIEW

**Comment by fabianfrz** [comment]
@fichtner something more to review ;)

File: source/manual/how-tos/nginx_basic_auth.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
+===========================================
+nginx: Basic Authentication & Authorization
+===========================================
+
+.. Warning::
+
+    Passwords in password files are not stored secure because this is not supported.
```
Comment (fichtner) [inline source/manual/how-tos/nginx_basic_auth.rst]: Cannot be stored securely (end sentence)

File: source/manual/how-tos/nginx_basic_auth.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    Your passwords are stored in plain text in the configuration and as md5 in the
+    nginx password files. Password hashes designed as such like bcrypt_, scrypt_ or
```
Comment (fichtner) [inline source/manual/how-tos/nginx_basic_auth.rst]: Secure password hashes like...

**Comment by fabianfrz** [comment]
done

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**